src/app/components/ItemPage/ItemPage.jsx

```diff
@@ -94,7 +94,7 @@ class ItemPage extends React.Component {
                 <Link
                   onClick={e => this.onClick(e, `${f.field}:"${obj['@id']}"`)}
```

Comment:
This is good but this string also needs to be updated in the `onClick` function. It's weird and probably not the best way to handle this, but the `to` attribute in the `Link` component is needed to know what the link will go to (for VO, for example). The `onClick` handler is used to pass the same url because it first needs to fetch data before it goes to that new page. So it needs to be updated in both locations.